src/test/java/org/jenkinsci/plugins/shortwspath/LocatorTest.java

```diff
@@ -54,8 +63,11 @@ public void doNothingIfThereIsEnoughRoom() throws Exception {
         FreeStyleProject p = f.createProject(FreeStyleProject.class, "and_a_project_in_it");
         p.setAssignedNode(s);
 
+        // Not enough for anything.
+        setMaxPathLength(s, 4096);
```

Comment:
In this test it should be set so it _is_ enough - comment suggests the opposite. Are you suggesting it is not on Windows (and that is why we need this)?